install/unix/source.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+<!-- EN-Revision: 64a010e59ce7e3c4cbbd72bbc307cfcce3d6e9bb Maintainer: zonuexe Status: ready -->
+<sect1 xml:id="install.unix.source" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>Unix および macOS システムでのソースコードからのインストール</title>
+ <para>
+  コンパイルに必要なソフトウェア:
+  <simplelist>
+   <member>
+    <link xlink:href="&url.gnu.make;">GNU <command>make</command></link>
+   </member>
+   <member>
+    C コンパイラ (PHP 8.0.0 以降は C99、PHP 8.4.0 以降は C11 互換性が必須)
+   </member>
+   <member>
+    Web サーバー
+   </member>
+   <member>
+    モジュールごとに必要なコンポーネント
+    (<acronym>GD</acronym>、 <acronym>PDF</acronym> ライブラリなど)
+   </member>
+  </simplelist>
+ </para>
+
+ <para>
+  Git ソースから直接ビルドする場合や、PHP のソースコードをカスタムしてビルドする場合、
+  以下の追加ツールが必要になる場合があります。
+  <itemizedlist>
+   <listitem>
+    <simpara>
+     <link xlink:href="&url.gnu.autoconf;">autoconf</link>:
+    </simpara>
+    <simplelist>
+     <member>
+      PHP 7.3 以降: バージョン 2.68 以上
+     </member>
+     <member>
+      PHP 7.2: バージョン 2.64 以上
+     </member>
+     <member>
+      PHP 7.1 以前: バージョン 2.59 以上
+     </member>
+    </simplelist>
+   </listitem>
+   <listitem>
+    <simpara>
+     <link xlink:href="&url.re2c;">re2c</link>:
+    </simpara>
+    <simplelist>
+     <member>
+      PHP 8.3 以降: バージョン 1.0.3 以上
+     </member>
+     <member>
+      PHP 8.2 以前: バージョン 0.13.4 以上
+     </member>
+    </simplelist>
+   </listitem>
+   <listitem>
+    <simpara>
+     <link xlink:href="&url.bison;">bison</link>:
+    </simpara>
+    <simplelist>
+     <member>
+      PHP 7.4 以降: バージョン 3.0.0 以上
+     </member>
+     <member>
+      PHP 7.3 以前: バージョン 2.4+ 以上 (3.x を含む)
+     </member>
+    </simplelist>
+   </listitem>
+  </itemizedlist>
+ </para>
+
+ <simpara>
+  PHP の初期セットアップと構成プロセスは、<command>configure</command>
+  スクリプトのコマンドライン オプションを使用して制御されます。
+  <command>./configure --help</command> を実行すると、
+  使用可能なオプションの一覧と簡単な説明が表示されます。
+  このマニュアルでは、さまざまなオプションを個別に説明しています。
+  <link linkend="configure.about">中心となる configure オプションのリストはこちら</link>
+  にありますが、さまざまな拡張機能ごとに固有のオプションについては
+  リファレンスページで説明されています。
+ </simpara>
+
+ <simpara>
+  構成スクリプトが実行された後、<command>make</command> コマンドを使用して PHP
+  をビルドできます。このマニュアルの<link linkend="install.problems">問題が起きた場合</link>
```

Comment:
This link is causing the build failure, should be:
`<link linkend="faq.installation">Installation section of the
  frequently asked questions</link>` if this is being synced with EN